static/form.js

```diff
@@ -9,6 +9,8 @@ function changeTestResultDisplay(show) {
   }
 }
 
+flatpickr(document.querySelectorAll('[type=date]'));
```

Comment:
Please put it in a different script tag so the other code will not fail when this resource is not loaded. For whatever reason ;)